guided-match/versions/v0.0.7.sql

```diff
@@ -0,0 +1,5 @@
+/*
+Version No Version Description
+v0.0.7     This file is a script to create add an additional column to error_messages */
+
+Alter table error_messages add column error_summary varchar(2000);
```

Comment:
Suggest we make both `error_message` and `error_summary` columns the same length - and somewhere in between 100 and 2000... there are some messages that are longer than 100 characters in the content spreadsheet - but none are anywhere near 2000.  1000 each for both?

Reply:
(and most are identical to each other barring the odd one or two and in those cases the text is very similar)

Reply:
sure no problem.I think it would be better to go longer. Do you want me to amend v0.0.7 or amend v0.0.6 and drop v0.0.7?

 It makes no difference to me.

Reply:
Me neither - amend v0.0.6 I guess as none of it is deployed anywhere yet anyway 👍

Reply:
> Me neither - amend v0.0.6 I guess as none of it is deployed anywhere yet anyway 👍
It is deployed everywhere as of 5 minutes ago.. :) But no problem - I will do destroys and recreates of everything anyway until we are ready to test properly on PROD (21st Sep)